com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -70,6 +70,21 @@ private enum SMAAStage
         readonly GraphicsFormat m_LutFormat;
         readonly HableCurve m_HableCurve;
 
+        //Viewport information
+
+        Vector2Int m_FullRes = new Vector2Int(1, 1);
+        Vector2Int m_DownsampledRes = new Vector2Int(1, 1);
+
+        private enum ResolutionGroup
+        {
+            Downsampled,
+            Full
+        }
+
+        private ResolutionGroup resGroup { set; get; }
+
+        private Vector2Int viewport { get { return resGroup == ResolutionGroup.Full ? m_FullRes : m_DownsampledRes; } }
```

Comment:
"viewport" might be a bit confusing.
In Unity terms, viewport.xy is the offset. Here we use it as a size so I first thought there was a mistake.
Maybe "viewportSize" ?